scripts/sct_download_data.py

```diff
@@ -54,6 +54,7 @@ def main(args=None):
 
     # Dictionary containing list of URLs for data names. Mirror servers are listed in order of priority.
     dict_url = {
+
```

Comment:
why this change?

Reply:
Hi, this was a dummy change in order to be able to create a PR in github ;)

Reply:
ah! i should learn to read commit messages...

Reply:
> if i understand correctly, the PAM50 binaries in [this repos](https://github.com/saulo-p/SCT-data) are version-tracked (i.e. are part of the git master tree) instead of being "packaged" within a release. 

In the dataset repo, the binaries are _both_ tracked and packaged. 
The packaging is done using the release assets, you can see for each release the original dataset `.zip` file.

> I thought we wanted to avoid this approach as it is not nice to pollute GH's server with binaries?  

The only problem I can see with the "scattered" binaries is that the dataset repo maintenance (cloning, pushing...) might be a little slow, but with the advantage that now the dataset files will be tracked which is usually good.

>   What is the "human" procedure for going with that solution? More specifically: when we create a new SCT release, what should be done?

The way I envision this, each dataset will have its own repository, so whenever there are changes to a particular dataset those would be committed to the respective repository.

As for the SCT releases, we could add some sort of script to SCT (replacing the functionality of `sct_download_data`) that points to the corresponding commits/releases in each dataset repository.